requirements/pip.txt

```diff
@@ -30,15 +30,11 @@ pyyaml==5.1.2
 Pygments==2.4.2
 
 # Basic tools
-# Redis 3.x has an incompatible change and fails
-# https://stackoverflow.com/questions/53331405/django-compress-error-invalid-input-of-type-cachekey
-# https://github.com/sebleier/django-redis-cache/pull/162
-redis==2.10.6  # pyup: ignore
-# Kombu >4.3 requires redis>=3.2
-kombu==4.3.0  # pyup: ignore
-# Celery 4.2 is incompatible with our code
-# when ALWAYS_EAGER = True
-celery==4.1.1  # pyup: ignore
+# redis==3.4.1 makes the celery workers to randomly get killed
```

Comment:
On what redis server version? I assume it's a mismatch between client & server.

Reply:
Engine Version Compatibility:
Version compatibility of the engine that will be run on your nodes
2.8.6